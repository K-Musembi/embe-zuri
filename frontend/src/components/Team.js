import React from "react";

const Team = () => {

    return (
        <div className="team-container">
            <h2>Meet the developer</h2>
            <div className="team-card">
                <div className="team-photo">
                    <img
                        src="/image-photo.jpg"  // photo in 'public folder'
                        alt="Dev"
                        className="photo"
                    />
                </div>
                <div className="team-info">
                    <h2>Kevin Musembi</h2>
                    <div className="title">Software Engineer</div>
                    <p>Software engineer with a passion for creating apps that have a societal impact. Musembi has 2 years experience in backend software development, and has worked on several projects. You can contact him through the email provided below.</p>
                    <p> Email: kevinmusembi.m@gmail.com</p>
                    <p> LinkedIn: <a href="https://www.linkedin.com/in/kevin-musembi/" target="_blank" rel="noopener noreferrer">Profile</a></p>
                </div>
            </div>
        </div>
    );
};

export default Team;
