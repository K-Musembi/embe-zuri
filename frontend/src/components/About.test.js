import React from 'react';
import renderer from 'react-test-renderer';
import Team from './About';

test('About component snapshot test', () => {
  const tree = renderer.create(<Team />).toJSON();
  expect(tree).toMatchSnapshot();
});
