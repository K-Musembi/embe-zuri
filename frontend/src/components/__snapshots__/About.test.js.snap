// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component snapshot test 1`] = `
<div
  className="about-container"
>
  <div
    className="about"
  >
    <img
      alt="Embe Zuri"
      src="/mango farm.jpeg"
    />
  </div>
  <div
    className="about-app-description"
  >
    <h1>
      About app
    </h1>
    <p>
      This application simplifies the process of diagnosing mango fruit tree diseases. The app uses an AI deep learning model to categorise leaf images based on signatures that differentiate between healthy mango leaves and those that are likely infected with a disease. The application can identify six common diseases tha affect mango trees and offer remedies and prevention measures to help farmers mitigate the diseases.
    </p>
  </div>
</div>
`;
